### ISSUE TITLE
Importing assets

### ISSUE THREAD

**Issue by losintikfos** [open-source-labs/vno#97]
Hi guys,

Could there be a specific setting I am missing in other to achieve scss imports within root `App.vue` as follows:

```
<style lang="scss">
@import "@/assets/scss/index.scss";
</style>
```
### Error:

```
[Scss compiler]:
 Syntax error within styles

 1  |  
    |  ^
 2  |  @import "assets/scss/index.scss";
    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 3  |  
```
### Structure:
```
myapp/
   assets/
       index.scss
    components/
    App.vue 
```
### Also Tried:
`@import ./assets/scss/index.scss;`

###  Config
```
{
  "root": "App",
  "entry": "./",
  "options": {
    "child": "Blogg",
    "port": "33033",
    "title": "myapp"
  }
}
```

**Comment by buttercubz** [comment]
importing with the scss plugin is currently not supported

**Comment by losintikfos** [comment]
Yeah it looks like it. IMHO importers plugin should be the top of the priority for `vno` reasons obvious.

**Comment by buttercubz** [comment]
> Yeah it looks like it. IMHO importers plugin should be the top of the priority for `vno` reasons obvious.

I agree, I will work on making this feature available soon

**Comment by losintikfos** [comment]
> > Yeah it looks like it. IMHO importers plugin should be the top of the priority for `vno` reasons obvious.
> 
> I agree, I will work on making this feature available soon

@buttercubz Do we have a timeline for this? we cannot wait to migrate our project to vno :+1:
